frontend/tests/features/collection/collection.test.ts

```diff
@@ -46,8 +46,8 @@ describe("Collection", () => {
       await createCollection({ collection: { name: collectionName }, page });
 
       // Try delete
-      await page.click(getTestID("collection-more-button"));
-      await page.click(getText("Delete Collection"));
+      await page.getByTestId("collection-more-button").click();
```

Comment:
thanks for changing this to use locators! 

I just researched a bit more to learn the difference between `page.click()` and `page.locator.click()`, the reason why Playwright recommends the latter is because `locator.click()` enforces strictness in the sense that if a locator returns multiple matches, Playwright will throw an error instead of just clicking the first match , which `page.click()` does, and could cause test flakiness as the UI evolves

https://github.com/microsoft/playwright/discussions/13649